Automattic/jetpack
# Podcasts: Refactor the audio player component

As part of the next phase of work on the podcasts features, we're
starting to develop a transcription block. This block needs an audio
player so we are refactoring the component used in the podcast block.

#### Changes proposed in this Pull Request:

This PR moves the component to a shared location. It refactors it to a functional component, and makes the necessary changes in the podcast player, so the audio player's state is handled by a prop.

It also adds skip forward and back buttons to the audio player interface

<img width="713" alt="Screen Shot 2021-01-07 at 13 42 15" src="https://user-images.githubusercontent.com/1699996/103937351-41b8ca00-50ee-11eb-9e52-cef60acc305e.png">


#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
Because this is a refactor, we need to check that the functionality hasn't changed/broken.

Add a podcast block to a post and check that it functions correctly, you can change episode, start and pause playback, etc.

#### Proposed changelog entry for your changes:

Podcast player block: add skip back/forward buttons
